### PR TITLE
Feature/add birth tests

### PR DESCRIPTION
**Pull request by aarongill1**

### CODE REVIEW

**Review by JamesWoodwardHMPO — APPROVED** [review]

File: config/index.js [hunk 1]
```diff
@@ -3,6 +3,10 @@
 const packageJson = require('../package.json');
 
 module.exports = {
+  env: process.env.NODE_ENV,
```
Comment (JamesWoodwardHMPO) [inline config/index.js:6]: Why have you added these, are they needed?

File: views/pages/birth/details.html [hunk 2]
```diff
@@ -27,76 +41,76 @@
...
             ],
             [
                 { text: "Name" },
-                { text: record.mother.forenames + " " + record.mother.surname }
+                { text: redact(record.mother.forenames + " " + record.mother.surname) }
             ],
             [
-                { text: "Maiden Name" },
+                { text: "Maiden name" },
                 { text: record.mother.maidenSurname }
```
[the remark this answers is not in the document]
Comment (aarongill1) [reply views/pages/birth/details.html]: Reverted

File: config/index.js (hunk 1 above)
Comment (aarongill1) [reply config/index.js]: Discussed and resolved

File: cypress/fixtures/example.json [hunk 3]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (JamesWoodwardHMPO) [inline cypress/fixtures/example.json:1]: We can delete this example file.

**Review by JamesWoodwardHMPO — DISMISSED** [review]

File: cypress/fixtures/example.json (hunk 3 above)
Comment (aarongill1) [reply cypress/fixtures/example.json:1]: Removed

**Review by JamesWoodwardHMPO — APPROVED** [review]

**Pull request merged**